docs/codeql/codeql-language-guides/codeql-library-for-csharp.rst

```diff
@@ -108,8 +108,8 @@ Count the number of lines of code, excluding the directory ``external``:
 .. code-block:: ql
 
    select sum(SourceFile f |
-     not exists(Folder external | external.getShortName() = "external" |
-                external.getAFolder*().getAFile() = f) |
+     not exists(Folder ext | ext.getShortName() = "external" |
```

Comment:
Slightly unrelated change, but the QL highlighter was highlighting the variable "external" as a keyword, so this was the easiest fix.

![image](https://user-images.githubusercontent.com/42641846/102998629-71c57200-451f-11eb-8e60-29722ff6469d.png)